runtime/doc/motion.txt

```diff
@@ -833,8 +822,8 @@ deletes the lines from the cursor position to mark 't'.  Hint: Use mark 't' for
 Top, 'b' for Bottom, etc..  Lowercase marks are restored when using undo and
 redo.
 
-Uppercase marks 'A to 'Z include the file name.  {Vi: no uppercase marks} You
-can use them to jump from file to file.  You can only use an uppercase mark
+Uppercase marks 'A to 'Z include the file name.
+You can use them to jump from file to file.  You can only use an uppercase mark
```

Comment:
I purposefully tried to minimize the diff on that paragraph, but I think you're right.